public/views/signin.html

```diff
@@ -1,76 +1,80 @@
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" landing>
+      <ul class="nav navbar-nav navbar-right">
+        <li class="hidden">
+          <a href="#page-top"></a>
+        </li>
+        <li class="page-scroll">
+          <a href="/">HOME</a>
+        </li>
+        <li class="page-scroll">
+          <a href="/#!/#about">ABOUT THE GAME</a>
+        </li>
+        <li class="page-scroll">
+          <a href="/#!/#charity">CHARITY</a>
+        </li>
+        </span>
+      </ul>
     </div>
+    <!-- /.navbar-collapse -->
   </div>
-<!-- Fixed Logo & Menu Bar End -->
-
-<!-- Landing Start -->
-<div class="wpb-teaser parallax-background" id="homer">
-  <div class="background-opacity"></div> 
-  <div class="modified-container">
-    <div class="row" id="main-text"> 
-    <!-- Teaser Text Start -->
-    <div id="landing-logo">
-      <h2 id="slogan">Sign in with:</h2>
-      <div id="login-buttons">
-        <a href="/auth/facebook"><img class="login-item" src="/img/icons/facebook.png"></a>
-        <a href="/auth/github"><img class="login-item" src="/img/icons/github.png"></a>
-        <a href="/auth/twitter"><img class="login-item" src="/img/icons/twitter.png"></a>
-        <a href="/auth/google"><img class="login-item" src="/img/icons/google.png"></a>
-      </div>
-      <p id="create">Or log in below:</p>
-
-      <div class="sign-body">
-        <div class="more-sign">
-          <div class="sign-error" ng-show="showError() === 'invalid'">
-            Please check your username/password and try again.
-          </div>
-          <form class="signup form-horizontal" action="/users/session" method="post">
-            <div class="control-group">
-              <label for="game" class="control-label"></label>
-              <div class="controls">
-                <input class="hide-game input-info" id="game" type="text" name="game" ng-model="gameId" placeholder="Game" novalidate />
-              </div>
-            </div>
-            <div class="control-group">
-              <label for="email" class="control-label"></label>
-              <div class="controls">
-                <input class="input-info" id="email" type="text" name="email" ng-model="showEmail" placeholder="Email" required novalidate />
-              </div>
+  <!-- /.container-fluid -->
+</nav>
+<div class="body" id="main-bod" ng-controller="IndexController">
+  <div class="signin-wrapper body">
+    <div class="row">
+      <div class="col-md-offset-4 col-md-4">
+        <div class=" row sign-up text-center">
+          <a href="#!/signup">
+          SIGN UP INSTEAD
+          </a>
+        </div>
+        <div class="row signin-sub-wrapper">
+          <div class="col-md-12 ">
+            <h4 class="signup-subtitle text-center">
+              Login to Play Game
+            </h4>
+            <div class="sign-error" ng-show="showError() === 'invalid'">
```

Comment:
Update `showError()` to simply return a boolean.

The less logic you have in your template, the better.